### ISSUE TITLE
Add Node and Deno in criteria to add a feature

### ISSUE THREAD

**Issue by regseb** [microsoft/TypeScript-DOM-lib-generator#1476]
The [addition](https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1444) of the [`static Response.json(data, init)`](https://fetch.spec.whatwg.org/#dom-response-json) method was [refused](https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1462#discussion_r1060729285) because it isn't implemented by two major vendors. It's implemented in Blink, but not in Gecko and WebKit.

It's already implemented in Node and Deno because it's mostly useful on the backend:

- Create easily a server response with JSON (cf. [Deno v1.22](https://deno.com/blog/v1.22#new-responsejson-static-method)):

  ```JavaScript
  import { serve } from "https://deno.land/std@0.165.0/http/server.ts";
  await serve(() => Response.json({ foo: "bar" }), { port: 8080 });
  ```

- Mock easily the response of a request:

  ```JavaScript
  sinon.stub(globalThis, "fetch").resolves(Response.json({ foo: "bar" }));
  ```

I think [Mozilla](https://bugzil.la/1758943) and [Apple](https://webkit.org/b/240375) are in no rush to implement this method. We may have this lack for a while.

---

I propose to add Node and Deno in the list of vendors.

> **[Why is my fancy API still not available here?](https://github.com/microsoft/TypeScript-DOM-lib-generator/blob/%40types/web%400.0.87/README.md#why-is-my-fancy-api-still-not-available-here)**
> A feature needs to be supported by two or more major browser engines or JavaScript runtime to be included here, to make sure there is a good consensus among vendors: __Gecko__ (Firefox), __Blink__ (Chrome/Edge), __WebKit__ (Safari), __Node__ and __Deno__.
>
> If the condition is met but still is not available here, first check the [contribution guidelines](#contribution-guidelines) below and then please [file an issue](https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/new).

**Comment by saschanaz** [comment]
I don't think this is the way to go since it's useless for browser purpose when only Blink/Node/Deno implements things and other browser engines do not.

Instead, I suggested #1402 for non-browser implementations so that we can have a set of lib for browsers and another lib for general JavaScript environments.
